src/raghvendra/ArrayMethod.java

```diff
@@ -0,0 +1,64 @@
+package raghvendra;
+
+public class ArrayMethod {
+	int min(int[] number) {
```

Comment:
Video sent on skype for this code, pls go through it.

Also Check this code for reference
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/318/files